database/models.js

```diff
@@ -0,0 +1,31 @@
+const db = require('./index.js');
+
+const { connection } = db;
+
+const query = (sql, holder, identifier) => {
```

Comment:
Importing bluebird might make your life easier when using promises, up to you though!